avocado/utils/linux_modules.py

```diff
@@ -40,7 +42,12 @@
 
 
 def load_module(module_name):
-    # Checks if a module has already been loaded
+    """
+    Checks if a module has already been loaded.
+    :param module_name: Name of module to check
+    :return: True if module is loaded, False otherwise
+    :rtype: Bool
+    """
```

Comment:
Since this is v3 already I'd also like to see it separated. I mentioned it optionally in v1 to avoid preventing it from merging, but since you already did so many rebases it'd make sense to actually go and split it. It's quite simple from cmdline, but I personally prefer `git cola` or `git citool` graphic tools to easily separate commits.